.gitignore

```diff
@@ -1,2 +1,4 @@
 .DS_Store
-node_modules
\ No newline at end of file
+node_modules
+package-lock.json
+.vscode
```

Comment:
We could probably create a base-level gitignore shared between all examples?

Reply:
This is one -- it's in the root directory of the entire project so it applies to all subdirectories.

Reply:
Ah right.